assessments/2022-12-05-strings/code.js

```diff
@@ -0,0 +1,31 @@
+const upToX = (s) => {
+  return s.substring(0, s.indexOf('x'))
+}
+
+const charactersAround = (s, n) => {
+  return s.substring(s[n] - 1) 
+}
+
+const middle = (s) => {
+  return s.substring(s.length / 4, s.length - s.length / 4)
+}
+
+const pair = (s1, s2) => {
+  return s1 + (' and ') + s2
+}
+
+const containsX = (s) => {
+  
+}
+
+const slug = (s1, s2, s3) => {
+  s1.toLowerCase() + s2.toLowerCase()('-') + ('-') + s3.toLowerCase
```

Comment:
This almost looks like you accidentally pasted something the wrong place. If you changed it like this it would be correct.

```suggestion
  s1.toLowerCase() + ('-')  + s2.toLowerCase() + ('-') + s3.toLowerCase
```

Though I'd also take out the unneeded parens:

```suggestion
  s1.toLowerCase() + '-'  + s2.toLowerCase() + '-' + s3.toLowerCase
```